rocky/rocky/templates/partials/ooi_detail_toolbar.html

```diff
@@ -2,16 +2,20 @@
 
 {% translate props.ooi_type as display_type %}
 <div class="horizontal-view toolbar">
-    {% if not ooi_past_due %}
-        {% if not ooi|is_finding and not ooi|is_finding_type %}
-            <a href="{% ooi_url "ooi_edit" ooi_id organization.code %}"
-               class="button ghost">{% blocktranslate %}Edit {{ display_type }}{% endblocktranslate %}</a>
-        {% endif %}
-        {% if not ooi|is_finding_type %}
-            <a href="{% ooi_url "ooi_delete" ooi_id organization.code %}"
-               class="button ghost">{% blocktranslate %}Delete {{ display_type }}{% endblocktranslate %}</a>
-        {% endif %}
+  <a href="{% ooi_url 'ooi_report' ooi_id organization.code query=mandatory_fields %}"
+     class="button">{% translate "Generate report" %}</a>
+  {% if not ooi_past_due %}
+    {% if ooi|is_finding and perms.tools.can_scan_organization %}
+      <a href="{% ooi_url 'finding_mute' ooi_id organization.code %}"
+         class="button">{% translate "Mute Finding" %}</a>
     {% endif %}
-    <a href="{% ooi_url "ooi_report" ooi_id organization.code query=mandatory_fields %}"
-       class="button ghost">{% translate "Generate report" %}</a>
+    {% if not ooi|is_finding and not ooi|is_finding_type %}
```

Comment:
This could simply be an `else` since it contradicts the condition on line 8?

Reply:
Here we check also for finding_type